src/customFileParser/main.ts

```diff
@@ -0,0 +1,81 @@
+import unpackGeometry from './unpackGeometry/main';
+import Sector from './../Sector';
+import countRenderedGeometries from './countRenderedGeometries';
+import { SectorMetadata, GeometryIndexHandler, TrueValueArrays, RenderedGeometryGroups }
+  from './sharedFileParserTypes';
+import CustomFileReader from './CustomFileReader';
+import { renderedGeometries, renderedGeometryToGroup } from './parserParameters';
+
+function fillSector(
```

Comment:
I guess the function shouldn't be called `fillSector` as that indicates fill an existing sector. What about createSector?